### PR TITLE
Nick

### PR DESCRIPTION
**Pull request by nrpriore**
leave game in top left
end turn via buttons
QGU timer
camera touch movement

### CODE REVIEW

File: Assets/Scripts/GameController.cs [hunk 1]
```diff
@@ -330,7 +371,7 @@ public class GameController : MonoBehaviour {
 
 	// Runs every frame
 	void Update() {
-		// Testing move/zoom
+		// Computer move/zoom
```
Comment (DaBernMon) [inline Assets/Scripts/GameController.cs:374]: If the updated move/zoom that works on phones now also works for click/drag on computers, this should probably be removed.

File: Assets/Scripts/GameController.cs [hunk 2]
```diff
@@ -368,6 +409,26 @@ public class GameController : MonoBehaviour {
...
+				CheckForTurn();
+				_qguTimer = 0f;
+			}
+		}
+	}
+
+	// Loads active games
+	private void CheckForTurn() {
+		if(!(bool)Server.QueryGames()["Success"]) {
+			Debug.Log("Query Games failed");
```
Comment (DaBernMon) [inline Assets/Scripts/GameController.cs:424]: If QG failed, should we return here?

File: Assets/Scripts/GameController.cs (hunk 1 above)
Comment (nrpriore) [reply Assets/Scripts/GameController.cs:374]: it won't :(
Mouse events will work on phone but not vice verca, and also each is optimized for its respective platform so I'd like to keep it separate

File: Assets/Scripts/GameController.cs (hunk 2 above)
Comment (nrpriore) [reply Assets/Scripts/GameController.cs:424]: We can, though the next line of code checks for UserTurn which will still be false if QGU failed and so nothing will happen anyways

**Review by DaBernMon — APPROVED** [review]

**Pull request merged**